tests/common.go

```diff
@@ -2763,6 +2763,55 @@ func DeleteBackupLocation(name string, backupLocationUID string, orgID string) e
 
 }
 
+// DeleteSchedule Delete Backup  Schedule
```

Comment:
nit: Fix comment please, follow same standards as comments above and remove extra space `// DeleteSchedule deletes backup schedule`

Reply:
Fixed it